### ISSUE TITLE
Isn't post-link.sh.Linux redundant now?

### ISSUE THREAD

**Issue by mingwandroid** [conda-forge/dbus-feedstock#9]
Hi @gqmelo, we shouldn't need post-link.sh.Linux anymore?

**Comment by gqmelo** [comment]
You are right, the post link script just copies the fallback files to ```$PREFIX/var/lib/dbus/machine-id```. As now dbus can find the fallback by itself we could remove the script.
